packages/core/package.json

```diff
@@ -22,9 +22,7 @@
     "@sentry/utils": "6.15.0",
     "tslib": "^1.9.3"
   },
-  "devDependencies": {
-    "typescript": "3.7.5"
-  },
+  "devDependencies": {},
```

Comment:
```suggestion
```
?

Reply:
Yeah this is because I just find and replaced the value. I can update.

Reply:
Actually I'm going to leave it like this. This makes makes the diff when adding new packages much cleaner (which in turn makes git blame easier to understand in the future)